cluster-scope/base/cert-manager.io/issuers/ingress-letsencrypt-production/issuer.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: ingress-letsencrypt-production
+  namespace: openshift-ingress
+spec:
+  acme:
+    email: ops@list.massopen.cloud
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: letsencrypt-production-key
+    solvers:
+      - selector:
+          dnsZones:
+            - "moc-infra.massopen.cloud"
+        dns01:
+          cnameStrategy: Follow
+          route53:
+            region: us-east-1
+            accessKeyID: AKIAYLUGMT7YARAARH6F
+            hostedZoneID: Z04486682XFINHTESP6B9
+            secretAccessKeySecretRef:
+              name: aws-route53-secret
+              key: aws_secret_access_key
```

We use completely different dns provider in this case. Please use:
```suggestion
          dnsZones:
            - "na.operate-first.cloud"
        dns01:
          cnameStrategy: Follow
        cloudDNS:
          project: aicoe-prow
          hostedZoneName: 'na-operate-first-cloud'
          serviceAccountSecretRef:
            key: sa
            name: gcp-sa
```